src/status.js

```diff
@@ -0,0 +1,17 @@
+import { saveTasksToLocalStorage, getTasks } from './app.js';
+
+export function updateStatus(index, completed) {
+  const tasks = getTasks();
+  if (tasks[index]) {
+    tasks[index].completed = completed;
+    saveTasksToLocalStorage();
+  }
+}
+
+export function toggleStatus(index) {
+  const tasks = getTasks();
+  if (tasks[index]) {
+    tasks[index].completed = !tasks[index].completed;
+    saveTasksToLocalStorage();
+  }
+}
```

Comment:
- [ OPTIONAL] These 2 methods have a lot in common, I implore you to combine them into one method. This will keep your code DRY and easy to mantain, update and debug. 😄

Reply:
@wayungi  Solved per your suggestion.